std/fmt/index.zig

```diff
@@ -892,9 +897,150 @@ test "fmt.format" {
     }
     {
         const value = @intToPtr(*i32, 0xdeadbeef);
+        // Print Pointer Path3
         try testFmt("pointer: i32@deadbeef\n", "pointer: {}\n", value);
         try testFmt("pointer: i32@deadbeef\n", "pointer: {*}\n", value);
     }
+    {
+        const value: [3]u8 = "abc";
+        try testFmt("const [3]u8: abc\n", "const [3]u8: {}\n", value);
```

Comment:
this is a redundant test. see line 884 above.

Reply:
Removing.